### PR TITLE
480p graphics fix in System Menu

### PR DESCRIPTION
**Pull request by Leseratte10**
This hack should work on 4.1 - 4.3 (normal Wii, would be useless on Mini). It implements the 480p fix in the System Menu, meaning, the System Menu itself (just the menu, no games or channels loaded through the menu) will include the 480p graphics patch that makes the graphics look better when the Wii is connected using Component cables. 

I don't have a component cable so I cannot test this, but maybe someone else has one.

### CODE REVIEW

File: priiloader/hacks_hash.ini [hunk 1]
```diff
@@ -314,3 +314,12 @@ minversion=1
 amount=1
 hash=0x68747470,0x733a2f2f,0x25732f6e,0x75732f73
 patch=0x6E756C6C
+[480p graphics fix in system menu]
+maxversion=4610
+minversion=448
```
Comment (DacoTaco) [inline priiloader/hacks_hash.ini]: you were only able to check versions >= 448?

Comment (Leseratte10) [reply priiloader/hacks_hash.ini]: Yeah, I only checked the code for 4.1+. Are 4.0 and below still used by many people? If needed, I could download and check earlier versions as well.

Comment (DacoTaco) [reply priiloader/hacks_hash.ini]: i don't think anyone uses anything < 4.1 either, was just wondering. the hash seems to be there in 2.0 as well. just not sure if its also the video init code (but i think it is)

Comment (Leseratte10) [reply priiloader/hacks_hash.ini]: If the whole hash is in other versions as well, it should definitely be the video init code. 
I do know that there are two different versions of the video init code, but if you found exactly that hash in 2.0 then apparently they always used the same one in the system menu. If it's present in 2.0, we could as well set minversion to 97.

Comment (DacoTaco) [reply priiloader/hacks_hash.ini]: then lower the min version, and i'll merge it.

**Comment by Leseratte10** [comment]
I've just set the min version to 1. It's done the same way for the other hacks, and nobody will have a pre-2.0 wii anyways. 
Although note that this hack hasn't been verified to work (yet). I don't have component cables so I can't test it. System Menu still works with that hack enabled at least.

**Comment by DacoTaco** [comment]
i could wait for the response on gbatemp i guess

**Comment by Leseratte10** [comment]
It's not that we'd need to hurry with merging that. You weren't planning to release 0.9 before the Wiimmfi patch is merged and tested anyways, so there's no harm in leaving this open until it's verified to work.

**Comment by gingerbeardman** [comment]
Hi. I just tested this and I can see a small difference (as expected).

ON


OFF

**Comment by DacoTaco** [comment]
i'll be honest and i see nearly no difference, but there is a difference.
does it look sharper too?

**Comment by gingerbeardman** [comment]
I have an second, older 480p TV that I used for my tests with Mario Kart Wii 480p fix. I'll check that one soon. The key is to find a type of image that shows the issue best. I'll look for a better place in System Menu.

The things is with Wii software, not all of it runs at the same resolution before it is scaled for output. I believe the System Menu runs at 686x456, not sure about Mario Kart Wii. But the point is that the effects of the 480p fix may be more/less in each software.

**Comment by gingerbeardman** [comment]
My 2nd, older 480p EDTV makes the difference more obvious.

You can pretty much make out the EPYX and 1985 in the bottom green line.

ON
ON TV2

OFF
OFF TV2

ORIGINAL SCREEN

**Comment by DacoTaco** [comment]
@Leseratte10 : good for merge then?

**Comment by Leseratte10** [comment]
Yeah.

**Comment by ross3695** [comment]
Is there any way to disable the deflicker filter also? It would be pretty cool if I could turn that off too and have an even sharper system menu, like I do with the rest of my system wads and stuff

**Comment by DacoTaco** [comment]
that has already been done, check #296

**Pull request merged**